service/worker/scanner/build_ids/scavenger.go

```diff
@@ -210,7 +211,12 @@ func (a *Activities) processNamespaceEntry(
 				return ctx.Err()
```

Comment:
Can probably remove this check now that we have the condition a few lines below.